.github/workflows/check.yml

```diff
@@ -25,3 +25,8 @@ jobs:
         run: make quiet
       - name: check-output
         run: ../tools/check-output.sh
+      - name: Upload output file
+        uses: actions/upload-artifact@v2
+        with:
+          name: output-log-file
+          path: source/std.pdf
```

Comment:
The name "output-log-file" seems to be inconsistent with the fact that we're uploading `source/std.pdf` which is an output, but not a log. `source/std.log` is what I would consider to be the log file. Maybe it would make more sense to call this `working-draft`?